de-lib/src/main/java/org/iplantc/de/collaborators/client/views/GroupViewImpl.java

```diff
@@ -122,10 +127,11 @@ public void onFailure(Throwable caught) {}
             @Override
             public void onSuccess(GroupDetailsDialog result) {
                 result.show();
```

Comment:
I was under the impression that launching new views/dialogs are done from presenter ?